### PR TITLE
stunting draft

### PR DESCRIPTION
**Pull request by Ninicorn**

### CODE REVIEW

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
++-------+-----------------------------------------+
+| LRI   | lower respiratory tract infection       |
++-------+-----------------------------------------+
+| MSLS  | measles                                 |
++-------+-----------------------------------------+
+
+
+.. note:: 
+   
+   Should we call this 2020 and have this under 2020 exposure? How do we deal with the different age groups?
```
Comment (aflaxman) [inline docs/source/gbd2019_models/risk_exposures/stunting/index.rst:63]: I think yes to call this 2020, and try to use the finer age groups from the 2020 results---vivarium should be able to do it!

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+   Should we call this 2020 and have this under 2020 exposure? How do we deal with the different age groups? 
+
+.. _stunting1.0:
+
+Risk Exposure Overview
+++++++++++++++++++++++
+
+Malnutrition is an imbalance between the body’s needs and its use and intake of nutrients. The imbalance can be caused by poor or lacking diet, poor hygiene, disease states, lack of knowledge, and cultural practices, among others. Underweight, stunting, wasting, obesity, and vitamin and mineral deficiencies are all forms of malnutrition. 
+
+**Chronic malnutrition**, also referred to as stunting ...
```
Comment (aflaxman) [inline docs/source/gbd2019_models/risk_exposures/stunting/index.rst:72]: Let's use that recent paper on communication strategies to guide how we talk about these concepts (while always staying compatible with GBD, of course...)

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+
+Risk Exposures Description in GBD
++++++++++++++++++++++++++++++++++
+
+.. _stunting1.1.1:
+
+Case definition
+---------------
+
+Stunting, a sub-compoonent indicator of child growth failure (CGF), is based on a categorical definition using the WHO 2006 growth standards for children 0-59 months. Definitions are based on Z-cores from the growth standards, which were derived from an international reference population. Mild, moderate and severe categorical prevalences were estimated for each of the three indicators. Theoretical minimum risk exposure level (TMREL) for stunting was assigned to be greater than or equal to one standard deviation below the mean (-1 SD) of the WHO 2006 standard weight-for-height curve. This has not changed since GBD 2010.
```
Comment (aflaxman) [inline docs/source/gbd2019_models/risk_exposures/stunting/index.rst:87]: typo "compoonent"

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+
+We also take the microdata sources and fit ensemble distributions to the shapes of the stunting, wasting, and underweight distributions. By doing this we can find characteristic shapes of stunting, wasting, and underweight curves. Once we have ST-GPR output as well as weights that define characteristic curve shapes, the last step is to combine them. We anchor the curves at the mean output from ST-GPR, use the curve shape from the ensemble distribution modeling, and then use an optimization function to find the standard deviation value that allows us to stretch/shrink the curve to best match the moderate and severe CGF estimates we got from ST-GPR. The final CGF estimates are the area under the curve for this optimized curve.
+
+
+
+.. note::
+  
+  Note that the z-score ranges from -7 to +7. If we limit ourselves to Z-scores between -4 and +4, we will be excluding a lot of kids.
+
+  In the paper that Ryan (GBD modeller for CGF and LWBSG) is working on right now, he presents the first results ever for "extreme" stunting which we define as kids with stunting Z scores below -4. For Ethiopia, that's about 7% of kids. So it's non-trivial!
```
Comment (aflaxman) [inline docs/source/gbd2019_models/risk_exposures/stunting/index.rst:128]: This makes me think we should include the extreme stunting category in our model, as well.

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+Vivarium Modeling Strategy
+++++++++++++++++++++++++++
+
+Child stunting in GBD has an **ordered polytomous variable**. It has rei_id 241
+
+We will use a 'propensity exposure model' for child stunting, in which each simulant is initialized with a “propensity” for a stunting category, and the simulant’s stunting category is determined by comparing this propensity to the overall stunting exposure/prevalence in the population. 
+
+.. todo::
+
+  Discuss with engineers if we want to use the propensity model or the invidivual Z-score distributions? Confirm with James how stunting was modelled in BEP.
```
Comment (aflaxman) [inline docs/source/gbd2019_models/risk_exposures/stunting/index.rst:245]: I suggest that we use an approach that allows us to include a relative risk for extreme stunting if GBD modelers can provide us with that value.  I think we also need an approach where we can correlate stunting and wasting (perhaps through the X Factor).

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst [hunk 6]
```diff
@@ -0,0 +1,337 @@
...
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+Validation Criteria
++++++++++++++++++++
+
+1. age, sex distribution of stunting prevalence categories matches GBD 2020
+2. age, sex, mean z-scores matches GBD 2020
+
```
Comment (aflaxman) [inline docs/source/gbd2019_models/risk_exposures/stunting/index.rst:334]: 3. correlation between stunting and wasting matches GBD 2020

should we also include relative risks here, or will that go in the Risk Effect model document?

**Review by aflaxman — APPROVED** [review]

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst (hunk 3 above)
Comment (beatrixkh) [reply docs/source/gbd2019_models/risk_exposures/stunting/index.rst:87]: also "Z-cores"

Comment (beatrixkh) [reply docs/source/gbd2019_models/risk_exposures/stunting/index.rst:87]: ^ I think it's worth mentioning which Z-score, here, to differentiate from other CGF categories. (Height for age, i think?)

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst [hunk 7]
```diff
@@ -0,0 +1,337 @@
...
+| Sev   |  < -3 Z score                        |
++-------+--------------------------------------+
+
+.. _stunting1.1.2:
+
+Input data
+----------
+
+Two types of input data are used in CGF estimation:  
+
```
Comment (beatrixkh) [inline docs/source/gbd2019_models/risk_exposures/stunting/index.rst:107]: typo "would may"

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst [hunk 8]
```diff
@@ -0,0 +1,337 @@
...
+  2. **Microdata**. This data does have individual anthropometric measurements. From these datasources, GBD can see entire distributions of CGF, while also collapsing them down to point prevalences like moderate and severe CGF. 
+
+
+.. _stunting1.1.3:
+
+Exposure estimation
+------------------- 
+
+In modeling CGF, all data types go into ST-GPR modeling. We have STGPR models for moderate, severe, and mean stunting, wasting, and underweight. The output of these STGPR models is an estimate of moderate, severe, and mean stunting, wasting, and underweight for all under 5 age groups, all locations, both sexes, and all years.
```
Comment (beatrixkh) [inline docs/source/gbd2019_models/risk_exposures/stunting/index.rst:118]: I think in other documentation we've tried to either cite the quote, or use "we" to refer to us on simulation science---maybe edit for clarity?

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst (hunk 4 above)
Comment (beatrixkh) [reply docs/source/gbd2019_models/risk_exposures/stunting/index.rst:128]: for documentation clarity: maybe add Ryan's last name, and date for "now" (Ryan Fitzgerald / june 2021)

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst [hunk 9]
```diff
@@ -0,0 +1,337 @@
...
+
+We also take the microdata sources and fit ensemble distributions to the shapes of the stunting, wasting, and underweight distributions. By doing this we can find characteristic shapes of stunting, wasting, and underweight curves. Once we have ST-GPR output as well as weights that define characteristic curve shapes, the last step is to combine them. We anchor the curves at the mean output from ST-GPR, use the curve shape from the ensemble distribution modeling, and then use an optimization function to find the standard deviation value that allows us to stretch/shrink the curve to best match the moderate and severe CGF estimates we got from ST-GPR. The final CGF estimates are the area under the curve for this optimized curve.
+
+
+
+.. note::
+  
+  Note that the z-score ranges from -7 to +7. If we limit ourselves to Z-scores between -4 and +4, we will be excluding a lot of kids.
```
Comment (beatrixkh) [inline docs/source/gbd2019_models/risk_exposures/stunting/index.rst:126]: If we only use -4 to +4, we should use the curves Ryan sent us to determine how many "a lot" is

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst [hunk 10]
```diff
@@ -0,0 +1,337 @@
...
+
+
+Data Description Tables
++++++++++++++++++++++++
+
+The following code can be used to access draw-level exposure data for the child_stunting risk factor, after additionally specifying desired location, age_group, and sex IDs. Exposure category cat 1,2,3 is severe, moderate and mild and exposure category cat4 is the TMREL.
+
+.. code-block:: python
+
+  get_draws(gbd_id_type='rei_id',
```
Comment (beatrixkh) [inline docs/source/gbd2019_models/risk_exposures/stunting/index.rst:296]: Do we want to pull this? We will need 2020, right?

Comment (beatrixkh) [reply docs/source/gbd2019_models/risk_exposures/stunting/index.rst:296]: oh nvm; i see the todo now

**Review by beatrixkh — APPROVED** [review]

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst (hunk 5 above)
Comment (Ninicorn) [reply docs/source/gbd2019_models/risk_exposures/stunting/index.rst:245]: hmm I dont think they have RRs for extreme stunting. I can double check.

File: docs/source/gbd2019_models/risk_exposures/stunting/index.rst (hunk 9 above)
Comment (Ninicorn) [reply docs/source/gbd2019_models/risk_exposures/stunting/index.rst]: ya...would be good to follow up with him on how his paper is developing. I don't think there are RRs for these extremes though.

**Pull request merged**